package.json

```diff
@@ -9,30 +9,60 @@
     "url": "https://github.com/zendeskgarden/tailwindcss/issues"
   },
   "version": "0.0.0",
-  "main": "src/index.js",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "build": "true",
+    "build": "rollup -c rollup.config.js",
     "format": "prettier-package-json --write && yarn format:all --write",
     "format:all": "prettier --loglevel warn '**/*.{js,ts,json,md}' '!CHANGELOG.md' '!dist/**'",
     "lint": "yarn lint:js && yarn lint:md",
     "lint:js": "eslint src/ --ext js,ts --max-warnings 0",
     "lint:md": "markdownlint README.md src/**/*.md",
     "prepare": "yarn build",
     "tag": "[ `git rev-parse --abbrev-ref HEAD` = 'master' ] && standard-version --no-verify",
-    "test": "prettier-package-json --list-different && yarn format:all --check && yarn lint"
+    "test": "prettier-package-json --list-different && yarn format:all --check && yarn lint && yarn test:js",
+    "test:js": "jest"
```

Comment:
This is flipped from what we're doing with other Garden repos. Typically:
- `yarn test` drives the typical local dev workflow (i.e. "jest")
- `yarn test:all` or `yarn test:ci` is the `&&`ed combination of the full test suite.

Reply:
Updated.